scripts/validate_docstrings.py

```diff
@@ -616,8 +621,22 @@ def validate_one(func_name):
                 errs.append(error('PR09', param_name=param))
 
     if doc.is_function_or_method:
-        if not doc.returns and 'return' in doc.method_source:
-            errs.append(error('RT01'))
+        if not doc.returns:
+            if "return" in doc.method_source:
+                errs.append(error('RT01'))
+        else:
+            if len(doc.returns) == 1 and doc.returns[0][1]:
+                errs.append(error('RT02'))
+            for name, type_, desc in doc.returns:
```

Comment:
What is the value of `name` in the case we've just got a single return?

I'd still like to have the names of the return fields that are failing in the returned error messages, and not give the user something like:
```
Return value has no description
Return value has no description
Return value has no description
```

@WillAyd can you propose your preferred solution for this?

For me the best would be the original one:
```
errs.append(error('RT03', return_name='{} '.format(name) if name else ''))
```

I think with the way it is implemented now it should be perfectly readable.

Reply:
when there is a single return and the first line is correctly written (ie no name is given), `name` contains the type, and `type_` is empty

Reply:
I'd prefer to rename `name` to `name_or_type` then, and possibly add a with what you just said.

Reply:
I'm totally indifferent as I really think this is an edge case. Whatever can be done simply and accurately works for me!